src/components/molecules/task/task.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Grid, Dropdown, Checkbox } from "semantic-ui-react";
+import {
+  StyledGridElipsisColumn,
+  StyledGridCheckboxColumn,
+  StyledGridTaskColumn,
+} from "./style/task.styled.component";
+
+export type ITask = {
+  taskName: String;
+  isTask?: boolean;
+  taskID: number;
+  onHandleDropDownChange: (e: any, data: any, taskID: number) => any;
```

Comment:
Is it possible to specify the actual types here instead of using "any"? It's better to do so because doing otherwise will be against the reason why we're using typescript in the first place. We should only use the "any" type only when it is absolutely necessary